torch/csrc/generic/Tensor.cpp

```diff
@@ -26,10 +29,8 @@
 #define COPY_FROM_ARRAY_CPU(ELTYPE, ARRAY, STORAGE, SIZE) \
```

Comment:
I think you can remove the `ELTYPE` from this macro now.

Reply:
Euh, forget what I said.... I'm wrong here :-)

Reply:
I had the same thought while looking at this, everything is just `void*` anyway, but the macro needs the same signature as the other guys I guess.